test/system/src/test/java/io/pravega/test/system/MultiReaderTxnWriterWithFailoverTest.java

```diff
@@ -305,7 +303,30 @@ public void multiReaderTxnWriterWithFailOverTest() throws Exception {
         log.info("Test {} succeeds ", "MultiReaderWriterTxnWithFailOver");
     }
 
-    private void performFailoverTest() throws InterruptedException {
+    private void closeReadersAndWriters() {
+        log.info("Closing writers");
+        writerList.forEach(writer -> {
+            try {
+                writer.close();
+            } catch (RetriesExhaustedException e) {
+                log.warn("Unable to close the client: ", e);
```

Comment:
Yes, we need to be strict. Created an issue for it. https://github.com/pravega/pravega/issues/1695